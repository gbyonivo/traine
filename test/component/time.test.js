import React from 'react';
import { create } from 'react-test-renderer';
import Time from '../../src/components/time';

const props = {
  date: '2018-07-21T00:35:00+01:00'
};

describe('Time', () => {
  it('should render correctly', () => {
    const actual = create(<Time {...props}/>);
    expect(actual).toMatchSnapshot();
  });
});