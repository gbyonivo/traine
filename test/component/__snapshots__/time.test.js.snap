// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Time should render correctly 1`] = `
<span>
  00:35
</span>
`;
